harbor-ui/src/routes/settings.rs

```diff
@@ -91,15 +61,20 @@ pub fn settings(harbor: &HarborWallet) -> Element<Message> {
             let button = h_button("Show Seed Words", SvgIcon::Eye, false)
                 .on_press(Message::ShowSeedWords(true));
 
+            let debug_stuff = if cfg!(debug_assertions) {
```

Comment:
If this is for users, this won't show

Reply:
yeah I want it hidden on non-debug builds it's just a handy place to put test widgets during dev